### ISSUE TITLE
Support Ruby 2.7

### ISSUE THREAD

**Issue by jcoyne** [railsconfig/config#259]
This causes problems as new_ostruct_member has been removed:
```
rails aborted!
NoMethodError: undefined method `new_ostruct_member' for #<Config::Options>
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/options.rb:177:in `method_missing'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/options.rb:204:in `block in __convert'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/options.rb:202:in `each'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/options.rb:202:in `__convert'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/options.rb:92:in `reload!'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config.rb:50:in `load_files'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config.rb:59:in `load_and_set_settings'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/integrations/rails/railtie.rb:11:in `preload'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4/lib/rails/railtie.rb:190:in `public_send'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4/lib/rails/railtie.rb:190:in `method_missing'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/config-2.0.0/lib/config/integrations/rails/railtie.rb:21:in `block in <class:Railtie>'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4/lib/active_support/lazy_load_hooks.rb:69:in `block in execute_hook'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4/lib/active_support/lazy_load_hooks.rb:62:in `with_execution_control'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4/lib/active_support/lazy_load_hooks.rb:67:in `execute_hook'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4/lib/active_support/lazy_load_hooks.rb:52:in `block in run_load_hooks'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4/lib/active_support/lazy_load_hooks.rb:51:in `each'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4/lib/active_support/lazy_load_hooks.rb:51:in `run_load_hooks'
/home/circleci/project/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4/lib/rails/application.rb:95:in `inherited'
```

**Comment by kapso** [comment]
Curious whats the release date for this fix? thanks.

**Comment by jcoyne** [comment]
@kapso this is an issue report, not a fix.

**Comment by kapso** [comment]
@jcoyne true, thanks for correcting me :)

**Comment by 9mm** [comment]
Also wondering

**Comment by aardvarkk** [comment]
Just ran into this myself.

**Comment by jcoyne** [comment]
Hey y'all, this is just an issue. The fix for the issue is in #255. However, that doesn't pass CI because Rails 4.2 doesn't like sprockets 3. That issue is fixed in #256. So, this problem is already solved, it just needs to be merged to master.

**Comment by Fryguy** [comment]
Fixed in #255 .  If you find any other Ruby 2.7 issues, please open more issues.  Thanks!

**Comment by pkuczynski** [comment]
Should be released in 2.2.0 before the end of the weekend.

**Comment by kapso** [comment]
@pkuczynski  Curious if the `2.2.0` is coming out anytime soon? Thanks.

**Comment by pkuczynski** [comment]
Working on this as we speak. Wanted to fit in #250, which is done and still working on #266 and #258. Hope to finish today :)

**Comment by pkuczynski** [comment]
Fix released as `2.2.1`
